mollie-payments-for-woocommerce/includes/mollie/wc/gateway/abstract.php

```diff
@@ -844,11 +870,233 @@ protected function handlePayedOrderWebhook( $order, $payment ) {
 
 	}
 
+	/**
+	 * @param WC_Order                     $order
+	 * @param Mollie\Api\Resources\Payment $payment
+	 */
+	protected function processRefunds( WC_Order $order, Mollie\Api\Resources\Payment $payment ) {
+
+		// Get order ID in the correct way depending on WooCommerce version
+		if ( version_compare( WC_VERSION, '3.0', '<' ) ) {
+			$order_id = $order->id;
+		} else {
+			$order_id = $order->get_id();
+		}
+
+		// Debug log ID (order id/payment id)
+		$log_id = 'order ' . $order_id . ' / payment ' . $payment->id;
+
+		// Add message to log
+		Mollie_WC_Plugin::debug( __METHOD__ . ' called for ' . $log_id );
+		
+		// Make sure there are refunds to process at all
+		if ( $payment->_links->refunds->href == null ) {
```

Comment:
Same comment as in `processChargebacks`

Reply:
https://github.com/mollie/WooCommerce/commit/ab5bd8873a30cc514ab291df51b8531dc0be8415